### PR TITLE
Lev 170

### PR DESCRIPTION
**Pull request by aarongill1**
Smoke tests that included login for 'real' authentication have been fixed. 

Cypress test reports with screenshots that can be viewed in the CI/CD pipeline.

### CODE REVIEW

File: Dockerfile-e2e [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM 113355358853.dkr.ecr.eu-west-1.amazonaws.com/ho/hmpo/lev/fro/cypress-testing:8c64f6e43a7e98ec1a25a1bf2bf7d830ef151e56
 # Use image below for local testing. Proxies also require removal.
-# FROM cypress/included:cypress-12.17.1-node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
+#FROM cypress/included:cypress-12.17.1-node-18.16.0-chrome-114.0.5735.133-1-ff-114.0.2-edge-114.0.1823.51-1
```
Comment (mehar-chauhan) [inline Dockerfile-e2e:3]: I appreciate that this was previously commented out, but is there a case where this is uncommented? If not, can this be removed?

File: cypress.config.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ module.exports = defineConfig({
   video: false,
   e2e: {
     setupNodeEvents(on, config) {
+      require('cypress-mochawesome-reporter/plugin')(on);
```
Comment (mehar-chauhan) [inline cypress.config.js:8]: `mochawesome` I love it

File: cypress/pages/LoginPage.js [hunk 3]
```diff
@@ -16,7 +16,6 @@ class LoginPage {
    */
   static login() {
     if (Cypress.env('e2e')) {
-      this.logout();
```
Comment (mehar-chauhan) [inline cypress/pages/LoginPage.js:19]: was there previously a flow in the old UI to logout, and now no longer exists?

File: cypress.config.js (hunk 2 above)
Comment (aarongill1) [reply cypress.config.js:8]: :)

File: cypress/pages/LoginPage.js (hunk 3 above)
Comment (aarongill1) [reply cypress/pages/LoginPage.js:19]: The original tests never worked, so this wasn't required. Also, the logout with newer KC now requires an id_token_hint that (used to be a static endpoint) so you can't log out anyway.  

TLDR - this was never required and doesn't add value.
